bundles/org.eclipse.equinox.p2.repository/src/org/eclipse/equinox/p2/repository/artifact/IArtifactDescriptor.java

```diff
@@ -81,6 +81,17 @@ public interface IArtifactDescriptor {
 	 */
 	@Deprecated String ARTIFACT_MD5 = "artifact.md5"; //$NON-NLS-1$
 
+	/**
+	 * An artifact descriptor property (value "aartifact.sha-256") indicating the
+	 * SHA256 checksum of the artifact bytes in its native format (after processing
+	 * steps have been applied).
+	 *
+	 * @since 2.4
+	 * @see #ARTIFACT_CHECKSUM
+	 */
+
+	String ARTIFACT_SHA256 = "artifact.sha-256"; //$NON-NLS-1$
```

Comment:
This one shouldn't be created. The new "dynamic" list of checksums is using ARTIFACT_CHECKSUM."algorytm" and there is even extension point to dyncmically produce new ones IIRC.

Reply:
@akurtakov yeah this was jsut more a guess as some tests are still referring to MD5 ... I'll try the other property now but tend to maybe revert this changes to the point where it is possible to disable algorithm but keep md5 for now as its a bit more work to adjust the test than initially thought and I can hardly guess if it not will not still have an impact.